### ISSUE TITLE
Two phase lottery needs better error handling

### ISSUE THREAD

**Issue by mgersh** [learning-unlimited/ESP-Website#1836]
Right now, if you try to register for a class, it'll check if you're in the right grade and silently drop your registration if you aren't. Ideally it would error more loudly.

**Comment by luac** [comment]
It's only supposed to display classes that are in the right grade, so this would only happen if there is a bug somewhere. But in general, yes, better error messaging would be good.

**Comment by mgersh** [comment]
Yeah, this came up because there was in fact a bug somewhere which was harder to debug because of this.

**Comment by pushkarkumar06** [comment]
Hi @willgearty ,

I’d like to work on this issue #1836  if it’s still open. Please let me know if I can take it.

Thanks!
